gis-data-sharing/open-data/index.md

```diff
@@ -6,16 +6,53 @@ author:
-- For data to be shared within the SGID, we prefer to share single feature type feature layers. This means, one data type and theme within a feature layer. This allows the greatest flexibility for the end users of the data.
-- Having an accurate title (Using Utah as a first identifier and spaces between words), summary and description are needed to allow users to discover your data.
-- For tagging data to be included within the ArcGIS Online SGID we start general and work to specific following the format `Agency/Department Name`, `SGID`, `ISO Category`, `other terms relevant to data usage`. We omit tags already contained in the layer title. Using the layer with the title `County Boundaries` as an example, we use `AGRC` and `SGID` as suggested tags. We chose `AGRC` since we want to include the agency and or department name. We chose `SGID` for inclusion in the SGID Open Data website. We omitted `County` and `Boundaries` since they are contained in the layer title. Please refer to [Using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more ideas and information.
-- Please make sure that the data is shared publicly with everyone, and include the appropriate SGID Group to share with. This will allow us to make your data available through the SGID Open Data website.
-- If you are the steward of the data and the data is considered authoritative, you can mark it as `Authoritative` so users will be able to tell at a glance that it is good data.
-- Please check the box to `Allow others to export to different formats` located at the bottom of the Settings page for the data to give the most flexibility for data usage.
+Does your department or division have authoritative datasets hosted in your own ArcGIS Online organization? Do you want to make them easy to find by anyone searching for Utah GIS Data? Share them on the [SGID Open Data website!](https://opendata.gis.utah.gov/) This makes them them easily discoverable while giving you full control over when they are updated.
+
+## Adding Your Data to Open Data
+
+All you need to do is **share your ArcGIS Online items with the appropriate SGID groups**. Items shared with these ArcGIS Online groups are automatically added to the SGID Open Data site.
+
+First, you have to be a member of the SGID groups your datasets best fit in. Navigate to AGRC's [groups page](https://utah.maps.arcgis.com/home/groups.html) and click on the appropriate group. Then click on "Join Group" to request access. We'll verify your request and add you to the group.
+
+Now that you're a member of the groups, share each item to the appropriate group. You also need to make sure the item is shared with "Everyone (public)" as well. Otherwise, it will only be visible to you (or other members of your organization if shared to the organization) in Open Data.
```

Comment:
```suggestion
Now that you're a member of the groups, share each item to the appropriate group. You also need to make sure the item is shared with "Everyone (public)". Otherwise, it will only be visible to you (or other members of your organization if shared to the organization) in Open Data.
```